rafiki/model/utils.py

```diff
@@ -51,7 +51,8 @@ def load_model_class(model_file_bytes, model_class, temp_mod_name=None) -> Type[
         raise InvalidModelClassError(e)
     finally:
         # Ensure that temp model file is removed upon model loading error
-        os.remove(temp_model_file_name)
+        # os.remove(temp_model_file_name)
```

Comment:
why do you comment out this line?

Reply:
Sorry, I forget to revoke the comments (in dev.py and utils.py) when committing. 
Comment out this line is for more convenient debugging.